Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -494,15 +496,15 @@ class StatusTableViewController: UITableViewController, UIGestureRecognizerDeleg
             case .Date:
                 cell.textLabel?.text = NSLocalizedString("Last Sentry", comment: "The title of the cell containing the last updated date")
 
-                if let date = dataManager.latestPumpStatus?.pumpDateComponents.date {
+                if let date = dataManager.latestPumpStatusDate {
```

Comment:
If this reflects the last date we read pump data from any source now, then the label on 497 should change.